CONTRIBUTING.md

```diff
@@ -0,0 +1,37 @@
+## Contributing
+
+[fork]: /fork
+[pr]: /compare
+[code-of-conduct]: CODE_OF_CONDUCT.md
+
+Hi there! We're thrilled that you'd like to contribute to this project. Your help is essential for keeping it great.
+
+## Issues and PRs
+
+If you have suggestions for how this project could be improved, or want to report a bug, open an issue! We'd love all and any contributions. If you have questions, too, we'd love to hear them.
+
+We'd also love PRs. If you're thinking of a large PR, we advise opening up an issue first to talk about it, though! Look at the links below if you're not sure how to open a PR.
+
+## Submitting a pull request
+
+1. [Fork][fork] and clone the repository.
+1. Configure and install the dependencies: `npm install`.
+1. Make sure the tests pass on your machine: `npm test`, note: these tests also apply the linter, so there's no need to lint separately.
```

Comment:
currently we do not have tests configured. Can you just change it to run `npm run lint` to lint code

Reply:
Done 👍

Reply:
Done. Changed the command here but kept adding test cases in point 5 as that is a standard best practice and I would encourage doing it.